src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,20 @@
-export const PostInfo = () => (
-  <>Put the post here</>
+import './PostInfo.scss';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+
+export const PostInfo = ({ post: { title, user, body, comments } }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{title}</h3>
+
+      <UserInfo user={user} />
```

Comment:
the `getUserById` function might return `null` theoretically. In this case, you will get an error (you will try to read properties from `null` inside the `UserInfo` component) - handle this case (`UserInfo` useless w/o `user` object)